packages/dd-trace/test/appsec/remote_config/manager.spec.js

```diff
@@ -149,28 +149,27 @@ describe('RemoteConfigManager', () => {
     })
   })
 
-  describe('on/off', () => {
+  describe('setProductHandler/removeProductHandler', () => {
     it('should update the product list and autostart or autostop', () => {
-      expect(rc.on('ASM_FEATURES', noop)).to.equal(rc)
+      rc.setProductHandler('ASM_FEATURES', noop)
 
       expect(rc.state.client.products).to.deep.equal(['ASM_FEATURES'])
       expect(rc.scheduler.start).to.have.been.calledOnce
 
-      rc.on('ASM_DATA', noop)
-      rc.on('ASM_DD', noop)
+      rc.setProductHandler('ASM_DATA', noop)
+      rc.setProductHandler('ASM_DD', noop)
 
       expect(rc.state.client.products).to.deep.equal(['ASM_FEATURES', 'ASM_DATA', 'ASM_DD'])
-      expect(rc.scheduler.start).to.have.been.calledThrice
```

Comment:
why remove that line ? can update it to .calledOnce instead ?

Reply:
Besides knowing that the timer has been started, this test shouldn't need to care about how many times it's called. That's an implementation detail.

All this test should care about is that a timer is running, which it knows because of the line `expect(rc.scheduler.start).to.have.been.calledOnce` above (which actually could just be `expect(rc.scheduler.start).to.have.been.called` since we don't care about the amount)

Reply:
Ah yeah sorry didn't look above, there is another check :+1: